src/graphql/mirror.js

```diff
@@ -318,6 +318,18 @@ export class Mirror {
           CREATE INDEX idx_connection_entries__connection_id
           ON connection_entries (connection_id)
         `,
+        dedent`\
+          CREATE TABLE network_log (
+              rowid INTEGER PRIMARY KEY,
+              query TEXT NOT NULL,
+              query_parameters TEXT NOT NULL,
```

Comment:
The term `query_parameters` is apt, but in other contexts can refer to
URL query parameters, like `page?foo=bar&baz=quux`. For clarity, please
specify that this is a JSON string:

```javascript
              query_parameters TEXT NOT NULL,  -- stringified JSON object
```

Reply:
updated